src/gui/advancedsettings.cpp

```diff
@@ -609,6 +617,23 @@ void AdvancedSettings::loadAdvancedSettings()
     // Announce to all tiers
     m_checkBoxAnnounceAllTiers.setChecked(session->announceToAllTiers());
     addRow(ANNOUNCE_ALL_TIERS, tr("Always announce to all tiers"), &m_checkBoxAnnounceAllTiers);
+
+    // Peer Turnover
+    m_spinBoxPeerTurnover.setMinimum(0);
+    m_spinBoxPeerTurnover.setMaximum(100);
+    m_spinBoxPeerTurnover.setValue(session->peerTurnover());
+    m_spinBoxPeerTurnover.setSuffix(tr(" %"));
+    addRow(PEER_TURNOVER, tr("Percentage of peers to disconnect"), &m_spinBoxPeerTurnover);
+    m_spinBoxPeerTurnoverCutoff.setMinimum(0);
+    m_spinBoxPeerTurnoverCutoff.setMaximum(100);
+    m_spinBoxPeerTurnoverCutoff.setSuffix(tr(" %"));
```

Comment:
It isn't clear to me that this tunable is a percentage. I think you should drop this line.

Reply:
It is still a [percentage](https://github.com/arvidn/libtorrent/blob/28f69ccbce9a3b860a65b6cbf6bcca247546ee45/src/session_impl.cpp#L3465).